src/controllers/dissolved-search/search.controller.ts

```diff
@@ -182,7 +182,7 @@ const previousNameResults = (company_name, company_number, date_of_cessation, da
             classes: "govuk-table__cell no-wrap"
         },
         {
-            text: formatPostCode(registered_office_address?.postal_code)
+            text: registered_office_address?.address_line_1 + " " + registered_office_address?.address_line_2 + " " + registered_office_address?.locality + " " + registered_office_address?.postal_code
```

Comment:
Won't this add redundant spaces if some values are not present? E.g. if address_line_1 is present and postcode wouldn't you end up with 2 additional spaces between them?
e.g.
Crown Way   CF14 3UZ

Reply:
Reverting to the postcode view as changes to the address are not in the scope of this ticket